### ISSUE TITLE
Playground fails permanently when Babylon.js 3.3 is selected

### ISSUE THREAD

**Issue by jure** [BabylonJS/Babylon.js#12336]
**Repro**
- Bug repro on [playground](https://playground.babylonjs.com): set the Babylon.js 3.3 version on any playground
- Expected result: Playground loads Babylon.js 3.3
- Current result: Playground shows white page and errors with `Cannot read properties of undefined (reading 'LastCreatedEngine')`. Additionally it prevents the page from loading ever again, until the cookies are cleared.

**Desktop:**
 - OS: MacOS
 - Browser Chrome 100

**Comment by RaananW** [comment]
As much as I am sure scenes will render in 3.3, it is a version that was released a few years ago. To support better features in the playground we introduced a few changes to the playground itself, which use features from babylon 4+, which prevents any version below 4 to work in the playground. (for example `LastCreatedEngine`). I will remove 3.3 from the list of supported rendering engines.

Temporarily, if you are stuck in version 3.3 reset the version and reload the playground. To reset call:

```javascript
localStorage.setItem("version", "")
```

In the console window of the playground (don't forget to reload).
